tests/unit_tests/ensemble_evaluator/test_ensemble_evaluator.py

```diff
@@ -12,11 +18,58 @@
     FORWARD_MODEL_STATE_RUNNING,
 )
 
-from .ensemble_evaluator_utils import send_dispatch_event
+from .ensemble_evaluator_utils import TestEnsemble, send_dispatch_event
 
 
-def test_new_monitor_can_pick_up_where_we_left_off(evaluator):
-    evaluator._start_running()
+@pytest.fixture(name="evaluator_handle")
+def start_ee_in_thread(make_ee_config):
+    msg_queue = queue.Queue()
+    ee_async = None
+
+    def _ee_start():
```

Comment:
Can we somehow change this function name to indicate that it is creating an event loop, and running the async function in it?
Maybe something like `_run_ee_async_in_new_eventloop`?